### DISCUSSION TITLE
Not working without internet

### DISCUSSION THREAD

**Discussion by WaqasBukhary** [open-webui/open-webui#6500] [General]
BUG
Open webui not working without internet giving following error in chrome log 
 index.ts:140 GET http://127.0.0.1:8080/ollama/api/version 500 (Internal Server Error)
 
Open WebUI Version: version: '0.3.32'
Ollama Version: 0.3.14 (got from command prompt 'ollama -v')
Operating System: [Windows 10]
Browser: Chrome Version 130.0.6723.70 (Official Build) (64-bit)

**Comment by rgaricano** [comment]
there is a open discussion about:  [offline usage #5759](https://github.com/open-webui/open-webui/discussions/5759)

**Comment by WaqasBukhary** [comment]
So it is security risk. Is it sending all my questions online? Does it have backdoors?
